aiohomekit/controller/abstract.py

```diff
@@ -109,11 +117,56 @@ def transport(self) -> Transport:
     def poll_interval(self) -> timedelta:
         """Returns how often the device should be polled."""
 
+    async def _process_disconnected_events(self):
+        pass
+
     def _async_description_update(
         self, description: AbstractDescription | None
     ) -> None:
+        if self.description != description:
+            logger.debug(
+                "%s: Description updated: old=%s new=%s",
+                self.id,
+                self.description,
+                description,
+            )
+
+        repopulate_accessories = False
+        if description:
+            if description.config_num > self.config_num:
```

Comment:
And then we can compare to self.config_num instead of the previous description, which fixes the problem I had with the previous patch.